lint/highlight.py

```diff
@@ -400,6 +433,28 @@ def draw(self, view):
                 region = sublime.Region(self.newlines[line], self.newlines[line])
                 gutter_regions[error_type].append(region)
 
+        def hide_phantoms(href):
+            view.erase_phantoms("sublimelinter")
+
+        for _line, _error in self.errors.items():
+            _column = _error[0][0]
```

Comment:
@rchl , it's my first time with python(i'm php dev)
i used the `_` prefix to be sure that those vars will not affect the rest of the code
i think that you can optimize it ;)